### PR TITLE
issue 346 transformed playbook.py into script instead of TDP cli

### LINKED ISSUE #346: [cli] Move the `playbooks` command to a script
The `tdp playbooks` command is used to generate meta playbooks, which are employed for deploying TDP without the necessity of using tdp-lib. We could convert it to a script instead of having it as a command.

The underlying question may be: "what's the purpose of tdp-lib?". Is it made for developers or for users?

### PR DESCRIPTION
**Pull request by SteBaum**
#### Which issue(s) this PR fixes

Fixes #346

#### Additional comments

- Removed the playbook command in main.py.
- Transformed playbook.py into a script a developer can use outside of tdp lib commands.
- Changed `import_playbook` to  `ansible.builtin.import_playbook` for output yml files.

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

File: scripts/playbooks.py [hunk 1]
```diff
@@ -121,3 +114,44 @@ def write_copyright_licence_headers(fd):
...
+        help="Only write operation from this collection, specify the collection name",
+    )
+    args = parser.parse_args()
+    services = args.services
+    output_dir = args.output_dir
+    for_collection = args.for_collection
+    collections = Collections.from_collection_list(
+        [
+            Collection.from_path(col)
+            for col in args.collection or TDP_COLLECTION_PATH.split(":")
```
Comment (PaulFarault) [inline scripts/playbooks.py:148]: You should use `os.pathsep` as a separator.

File: scripts/playbooks.py [hunk 2]
```diff
@@ -1,37 +1,28 @@
...
-    type=str,
-    help="Only write operation from this collection",
-    required=False,
-    multiple=True,
-)
-@collections
-def playbooks(services, output_dir, for_collection, collections):
+
+def playbooks(
+    collections: Collections,
```
Comment (PaulFarault) [inline scripts/playbooks.py:20]: Add a condition before calling playbooks if you don't want `collections` to be empty.

File: scripts/playbooks.py
Comment (PaulFarault) [inline scripts/playbooks.py:1]: Would be nice to add a docstring describing the purpose of this script

**Review by PaulFarault — CHANGES_REQUESTED** [review]

**Comment by PaulFarault** [comment]
We could leverage click's features to parse options (and use the ones defined for the cli). This doesn't mean that the command will is part of the CLI.

**Review by PaulFarault — DISMISSED** [review]
We could leverage click's features to parse options (and use the ones defined for the cli). This doesn't mean that the command will is part of the CLI.

**Review by PaulFarault — CHANGES_REQUESTED** [review]

**Comment by SteBaum** [comment]
I have changed the PR. It now does what is described in the description above and does not change failure if the meta outputdirectory already exists.

**Review by rpignolet — APPROVED** [review]

**Pull request merged**